docs/advanced-features/measuring-performance.md

```diff
@@ -181,6 +181,42 @@ export function reportWebVitals(metric) {
 >
 > Read more about [sending results to Google Analytics](https://github.com/GoogleChrome/web-vitals#send-the-results-to-google-analytics).
 
+## Web Vitals Attribution
+
+Web Vitals are useful for gathering the numbers and tracking improvements or regressions, but the numbers alone are not very useful for debugging performance issue.
```

Comment:
This needs a bit of work - blocking so we can iterate on it. 

In general, prefer to speak to the positive instead, rather than negative. Also, I think we can remove the quotes.

Reply:
updated the docs, PTAL